Common Collections/Strings/Process Some Strings/task.md

```diff
@@ -0,0 +1,11 @@
+## Process Some Strings
+
+There's tons of useful standard library functions for strings. Let's try and use some of
+them!
+
+<div class="hint">You may or may not need to convert from <code>&str</code> to <code>String</code>.</div>
+
+<div class="hint">For the <code>compose_me</code> method: You can either use the <code>format!</code> macro, or convert the string
```

Comment:
<div class="hint">For the <code>compose_me</code> method: You can either use the <code>format!</code> macro or convert the string